.github/workflows/tests.yml

```diff
@@ -56,6 +56,16 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox installation
+        run: |
+          bash
```

Comment:
Hi there!

I don't think this line does anything. Everything already runs on bash anyway. So:

```suggestion
```

If you do need to specify it in the future, you can instead write:

```
 - name: SpinalCord Toolbox installation
   shell: bash
   run: |
     ....
```
Docs: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell